pubsub/index.js

```diff
@@ -6,239 +6,172 @@
 
 const async = require('async')
 const crypto = require('crypto')
+const equal = require('fast-deep-equal');
```

Comment:
Could we call this deepEqual, it'd be a bit more obvious how it's doing the comparison.

Reply:
The linter is complaining about this semi-colon.